src/components/Loader.jsx

```diff
@@ -0,0 +1,5 @@
+import React from "react";
+
+export const Loader = () => {
+  return <div>Loader</div>;
```

Comment:
probably more natural would be to do : <div>...loading</div>

Reply:
done